packages/toggleInput/components/ToggleInput.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from "react";
+import { cx } from "emotion";
+
+import {
+  flex,
+  flexItem,
+  padding,
+  visuallyHidden,
+  display,
+  tintContent,
+  tintContentSecondary
+} from "../../shared/styles/styleUtils";
+import { error, success } from "../../design-tokens/build/js/designTokens";
+import { CheckboxInputProps } from "../../checkboxInput/components/CheckboxInput";
+
+export enum ToggleInputAppearance {
```

Comment:
Will this use the shared `InputAppearance` enum from the select component PR once merged?

Reply:
of course :)